### PR TITLE
Insex update

### PR DESCRIPTION
**Pull request by nocrad349**
Added some URLs to insex.yml. 
I've tested it on a few scenes for each added URLs and didn't see any problems. 

insex.yml passes the project's validation.

### CODE REVIEW

**Comment by bnkai** [comment]
@nocrad349 as mentioned in the other PR #338 i am including that scraper in here
did a quick conversion and an image fix

```yaml
name: insex
sceneByURL:
  - action: scrapeXPath
    url:
      - hardtied.com/ht
      - infernalrestraints.com/ir
      - insex.com/is
      - insexondemand.com/iod
      - realtimebondage.com/rtb
      - sexuallybroken.com/sb
    scraper: sceneScraper
  - action: scrapeXPath
    url:
      - topgrl.com/tg
    scraper: topgrlScraper

xPathScrapers:
  sceneScraper:
    common: &commonSelectors
      $studio: //title/text()
    scene:
      Title: &titleSelector //div[@class="is-size-5 has-text-weight-bold"]/text()
      Date:
        selector: &dateSelector //span[@class="tag is-dark"]/text()
      Details:
        selector: //div[@class="is-size-7 has-text-white-ter"]/preceding-sibling::div[1]/text()
        concat: "<br>"
        postProcess: &ppDetailsAttr
          - replace:
              - regex: <br>
                with: "\n"
      Performers: &performersAttr
        Name: //div[@class="is-size-6 has-text-white-ter"]/a[@class="tag is-dark"]/text()
      Tags: &tagsAttr
        Name: //div[@class="is-size-7 has-text-white-ter"]/following-sibling::div/span/text()
      Studio:
        Name: $studio
      Image: &imageSelector //video-js[@id="videoPlayer" or @id="iodvideo"]/@poster

  topgrlScraper:
    common: *commonSelectors
    scene:
      Title:
        selector: *titleSelector
        # Topgrl site doesnt receive any updates anymore and lots of scene's have "[Archive]"  in the title.
        # No idea what makes those scene's different, but I'm assuming most people don't want to add that to the title in Stash.
        postProcess:
          - replace:
              - regex: (\[Archive\] )+(.*)
                with: $2
      Date:
        selector: *dateSelector
      Details:
        selector: //div[@class="is-size-5 has-text-weight-bold"]/following-sibling::div[3]/text()
        concat: "<br>"
        postProcess: *ppDetailsAttr

      Performers: *performersAttr
      Studio:
        Name: $studio
      Image: *imageSelector
# Last Updated January 15, 2021
```
i'll test later with more scenes , can you close the other PR and include the above and the relevant scrapers list addition here?

**Comment by nocrad349** [comment]
I'll look at it later today or tomorrow.

I'm also working on 2 other sites that belong to insex/intersec (sensualpain.com and paintoy.com). They still use an old layout, so none of the selectors match. Should I also include that in insex.yml as they all belong to a single company?

One of those sites (sensualpain.com) sometimes includes tags in the text description, just as a list of comma separated words. Only way I see to recognize those is assuming that a sentence with like 7 comma's or more is a tag list. But that doesn't always work, resulting in a regular sentence being recognised as tags or the other way around. What do I do, include code that knowingly fails on occasion or ignore tags altogether?

**Comment by bnkai** [comment]
For sites with different layout i would use a separate file

For tags or other meta in the description imho its a waste of effort and time.Just leave them out

**Comment by nocrad349** [comment]
@bnkai I've closed the topgrl PR and included your version in this PR. I have tested it with a few dozen scenes and everything seems to work.

**Comment by nocrad349** [comment]
Added some more sites from insex/intersec.
I think we now have all current insex/intersec sites.

File: scrapers/insex.yml [hunk 1]
```diff
@@ -2,31 +2,82 @@ name: insex
...
         selector: //div[@class="is-size-7 has-text-white-ter"]/preceding-sibling::div[1]/text()
         concat: "<br>"
-        postProcess:
+        postProcess: &ppDetailsAttr
           - replace:
-            - regex: <br>
-              with: "\n"
-      Performers:
+              - regex: <br>
+                with: "\n"
```
Comment (bnkai) [inline scrapers/insex.yml:35]: The formatting of the descritpion imo looks more accurate if 
```yaml
                with: "\n\n"
```
is used

File: scrapers/insexOL.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        selector: //span[@class="articlePostDateText"][1]/text()
+        postProcess:
+          - parseDate: "January 2, 2006"
+      Details:
+        selector: //td[@class="articleCopyText"]/text()
+        concat: "<br>"
+        postProcess:
+          - replace:
+            - regex: <br>
+              with: "\n"
```
Comment (bnkai) [inline scrapers/insexOL.yml:23]: same as above 
```yaml
              with: "\n\n"
```

**Comment by nocrad349** [comment]
Separating the text-nodes with \n\n does make sense, on the sites a double <br> is used way more often than a single one. 

I've also simplified it slighty, instead of inserting '<br>' and then replacing it with \n\n, the \n\n is now inserted directly.

**Pull request merged**